emscripten/ARToolKitJS.cpp

```diff
@@ -139,6 +144,14 @@ extern "C" {
 
 		if (arc->detectedPage >= 0) {
 			int trackResult = ar2TrackingMod(arc->ar2Handle, arc->surfaceSet[arc->detectedPage], arc->videoFrame, trans, &err);
+			for (int j = 0; j < 3; j++) {
+				for (int k = 0; k < 4; k++) {
+					transF[j][k] = trans[j][k];
+				}
+			}
+			if (arFilterTransMat(arc->ftmi, transF, 1) < 0) {
```

Comment:
the third argument of  arFilterTransMat should a value that is zero or 1  as explained in the doc http://www.artoolkitx.org/artoolkit5/doc/apiref/arFilterTransMat_h/index.html#//apple_ref/c/func/arFilterTransMat